src/opnsense/mvc/app/library/OPNsense/Auth/Voucher.php

```diff
@@ -237,20 +241,22 @@ public function generateVouchers($vouchergroup, $count, $validity, $starttime =
                     // save user, hash password first
                     $generatedPasswordHash = crypt($generatedPassword, '$6$');
                     $stmt = $this->dbHandle->prepare('
-                                insert into vouchers(username, password, vouchergroup, validity, starttime)
-                                values (:username, :password, :vouchergroup, :validity, :starttime)
+                                insert into vouchers(username, password, vouchergroup, validity, expirytime, starttime)
+                                values (:username, :password, :vouchergroup, :validity, :expirytime,:starttime)
```

Comment:
add a space after the comma

Reply:
hi - just a numb question: how to find Squash and merge? :D 
this is my first one - so I dunno...

Reply:
This can be done on GitHub using the button for that. The alternative is doing it on the shell using an interactive rebase.

Reply:
we can do squash with final merge, but it would be better to add all commits to the same PR, you can just update your git PR branch and GitHub will update the PR

Reply:
ok, seems like I can't do that without writepermissions...
probably it's not what I wanted... the aim is to have all 3 pullrequests in one

Reply:
@Stephanowicz I would recommend you to merge all your branches together. When @fichtner will merge it squashed, all merge commits are gone anyway.
If you want to improve your git skills, you can try `git rebase -i master` on that branch.

Reply:
no worries, take your time to sort this out. it should be possible when all commits are on the same branch and pushed